packages/hyperion-autologging/src/ALSurfaceProxy.ts

```diff
@@ -35,25 +34,27 @@ type ProxyInitOptions =
  * If we can find a way around this limitation, we can use a simpler logic
  * like the following:
  */
-function SurfaceProxy(props: React.PropsWithChildren<ProxyInitOptions>) {
+function SurfaceProxy(props: React.PropsWithChildren<ProxyInitOptions>): React.ReactElement | React.ReactNode {
```

Comment:
You can instead use the following: 
`ReactModule.createElement(React.Fragment, {}, children);`

That is the same as using `<>{childern}</>` in the jsx files.

Reply:
Yeah this is what I was just starting to look into,  thanks for the tip!